src/com/magento/idea/magento2plugin/actions/generation/generator/ModuleGraphQlResolverClassGenerator.java

```diff
@@ -0,0 +1,130 @@
+                    GraphQlResolverPhp.GRAPHQL_RESOLVER_TEMPLATE_NAME, attributes, project);
+
+            graphQlFile[0] = graphQlResolverClass.getContainingFile();
+            CodeStyleSettings codeStyleSettings = new CodeStyleSettings((PhpFile) graphQlFile[0]);
+            codeStyleSettings.adjustBeforeWrite();
+
+            PsiDocumentManager psiDocumentManager = PsiDocumentManager.getInstance(project);
+            Document document = psiDocumentManager.getDocument(graphQlFile[0]);
+            int insertPos = getInsertPos(graphQlResolverClass);
+            document.insertString(insertPos, methodTemplate);
+            int endPos = insertPos + methodTemplate.length() + 1;
+            CodeStyleManager.getInstance(project).reformatText(graphQlFile[0], insertPos, endPos);
+            psiDocumentManager.commitDocument(document);
+            codeStyleSettings.restore();
+        });
+        return graphQlFile[0];
+    }
+
+    private int getInsertPos(PhpClass graphQlResolverClass) {
+        int insertPos = -1;
+        LeafPsiElement[] leafElements = PsiTreeUtil.getChildrenOfType(graphQlResolverClass, LeafPsiElement.class);
+        for (LeafPsiElement leafPsiElement: leafElements) {
+            if (!leafPsiElement.getText().equals(MagentoPhpClass.CLOSING_TAG)) {
+                continue;
+            }
+            insertPos = leafPsiElement.getTextOffset();
+        }
+        return insertPos;
+    }
+
+    private PhpClass createGraphQlResolverClass(String actionName) {
+        PsiDirectory parentDirectory = ModuleIndex.getInstance(project)
+                .getModuleDirectoryByModuleName(graphQlResolverFileData.getGraphQlResolverModule());
+        String[] graphQlResolverDirectories = graphQlResolverFileData.getGraphQlResolverDirectory().split(File.separator);
+        for (String graphQlResolverDirectory: graphQlResolverDirectories) {
+            parentDirectory = directoryGenerator.findOrCreateSubdirectory(parentDirectory, graphQlResolverDirectory);
+        }
+
+        Properties attributes = getAttributes();
+        PsiFile graphQlResolverFile = fileFromTemplateGenerator.generate(
+                GraphQlResolverPhp.getInstance(graphQlResolverFileData.getGraphQlResolverClassName()),
+                attributes,
+                parentDirectory,
+                actionName
+        );
+        if (graphQlResolverFile == null) {
+            return null;
+        }
+        return getFirstClassOfFile.execute((PhpFile) graphQlResolverFile);
+    }
+
+    protected void fillAttributes(Properties attributes) {
+        String graphQlResolverClassName = graphQlResolverFileData.getGraphQlResolverClassName();
+        attributes.setProperty("NAME", graphQlResolverClassName);
+        attributes.setProperty("NAMESPACE", graphQlResolverFileData.getNamespace());
+        attributes.setProperty("USE", GraphQlResolverPhp.INTERFACE_FQN);
+        attributes.setProperty("IMPLEMENTS", GraphQlResolverPhp.INTERFACE_NAME);
```

Comment:
Why do we need USE and IMPLEMENTS attributes If they are hardcoded in the template?

Reply:
I removed these attributes because of these attributes became unnecessary after I add the template.
Thanks for you review!